### ISSUE TITLE
src clean up

### ISSUE THREAD

**Issue by atherdon** [GroceriStar/food-datasets-csv-parser#249]
**Is your feature request related to a problem? Please describe.**
@EbrahimKreem just asking your opinion, but it will be a task for our new teammates.
what is your opinion about code inside `fileSystems`, `mainWrapper.js` and `index.js`?
I think we can clean it up, especially after our latest changes. curious to hear your opinion.

**Comment by EbrahimKreem** [comment]
yeah totally agree with you because `fileSystem.js` and `mainWrapper.js`  are useless in projects3.0.

about `index.js` do you mean that it should be removed or imports for `fileSystem.js` and `mainWrapper` should be removed?

**Comment by EbrahimKreem** [comment]
I think that `utils.js` can be removed also

**Comment by atherdon** [comment]
about `index.js` - i'm just not sure if we should export all of these
methods, because we're using only few in projects3.0
and as we'll remove stuff from other files - it will be cleaned up a bit

>
